src/battle-tooltips.ts

```diff
@@ -133,6 +133,12 @@ class ModifiableValue {
 	}
 }
 
+interface AbilityData {
```

Comment:
I honestly wouldn't bother naming an interface that's used in exactly one function's return value, though.

Reply:
Good point. I dont even think you need to write the interface at all? TS should be able to infer it based on the `abilityData` variable shape, so you dont even need to write out the type anywhere.